bin/run-test.sh

```diff
@@ -50,5 +50,3 @@ fi
 
 # Run non-interference tests
 node test/non-interference/http-e2e.js || exit 1
-node test/non-interference/express-e2e.js || exit 1
```

Comment:
These never ran on anything other than 0.12. I thought it would be more misleading to leave them if they can never run. At some point we should write updated versions of these.